tests/single/bundle/test_storage_rules.py

```diff
@@ -89,130 +88,142 @@ def build_userop_for_factory(w3, entrypoint_contract, factory_contract, rule):
 
 STAKED = True
 UNSTAKED = False
-PAYMASTER = "TestRulesPaymaster"
-FACTORY = "TestRulesFactory"
-SENDER = "TestRulesAccount"
-AGGREGATOR = "TestRulesAggregator"
+PAYMASTER = "paymaster"
+FACTORY = "factory"
+SENDER = "account"
+AGGREGATOR = "aggregator"
 
-StorageTestCase = collections.namedtuple(
-    "StorageTestCase",
-    ["ruleId", "rule", "staked", "entity", "userop_build_func", "assert_func"],
+
+def entity_to_contract_name(entity):
+    if entity == PAYMASTER:
+        return "TestRulesPaymaster"
+    elif entity == FACTORY:
+        return "TestRulesFactory"
+    elif entity == SENDER:
+        return "TestRulesAccount"
+    elif entity == AGGREGATOR:
+        return "TestRulesAggregator"
+
+
+ValidationRuleTestCase = collections.namedtuple(
```

Comment:
Are you sure about piling this rename into this PR? Maybe we can have a separate PR just with the renaming first?

Reply:
Since it's only a rename and I use that separation in the new 7560 tests, I think it's ok. There aren't any other places, besides `test_storage_rules.py` that use that named tuple